types/lazy.js/lazy.js-tests.ts

```diff
@@ -231,6 +247,10 @@ stringSeq = stringSeq.toLowerCase();
 stringSeq = stringSeq.toUpperCase();
 
 // flatten
-var fooSeqSeqSequence: LazyJS.Sequence<LazyJS.Sequence<LazyJS.Sequence<Foo>>>;
+declare var fooSeqSeqSequence: LazyJS.Sequence<LazyJS.Sequence<LazyJS.Sequence<Foo>>>;
 fooSequence = fooSeqSeqSequence.flatten();
 fooSequence = fooSeqSeqSequence.flatten(true).flatten(true);
+
+// Iterator
+
+new LazyJS.Iterator(fooSequence);
```

Comment:
I tried to run this code and it didn't work; does this actually exist on the top level export? Or am I missing something about how to use this particular API?

(Since nobody else reviewed it, I get the pleasure 😅 )

Reply:
As it turns out, no, it doesn't exist. But for some reason the lazy.js documentation says it does. I made the mistake of assuming the documentation was correct.

I'll push a change to remove this tomorrow.

Reply:
Done.